tutorials/customizing-base-map-styles/index.md

```diff
@@ -0,0 +1,332 @@
+ - gamma: Gamma correction of an element (0.01 and 10.0)
+ - invert_lightness: inverts the existing lightness
+ - visibility: Changes visibility options for an element (on, off, or simplified)
+ - color: Sets the color of an element (using RGB Hex Strings)
+ - weight: Sets the weight of a feature in pixels
+
+
+## Changing map elements
+
+The Google Maps API (and by extension, Earth Engine) gives you the ability to control a huge number of map features and elements.
+
+The full list of elements that you can modify can be found in the Google Maps documentation: https://developers.google.com/maps/documentation/javascript/style-reference
+
+The full list of features (also in the Google Maps API documentation linked above) includes geometries and labels (including their texts and icons). All styler options work with each of these features.
+
+Let's consider a couple of examples to edit other elements and features
+
+```javascript
+
+//Remove icons
+var iconChange = [{ // Change map saturation.
+  "stylers": [{
+    "gamma": .2
+  }]
+}, { // Change label properties.
+  "elementType": "labels",
+  "stylers": [{
+    "visibility": "off",
+    "color": "#000055"
+  }]
+}, { // Change road properties.
+  "featureType": "road",
+  "elementType": "geometry",
+  "stylers": [{
+    "visibility": "off",
+    "color": "#000055"
+  }]
+}, { // Change road labels
+  "featureType": "road",
+  "elementType": "labels",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, { // Change icon properties
+  "elementType": "labels.icon",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, { // Change POI options
+  "featureType": "poi",
+  "elementType": "all",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, {
+  "featureType": "administrative",
+  "elementType": "geometry.fill",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}, {
+  "featureType": "administrative",
+  "elementType": "geometry.stroke",
+  "stylers": [{
+    "visibility": "off"
+  }]
+}];
+
+
+//Enhanced road network visualization
+
+var roadNetwork = [{
+    "stylers": [{
+      "saturation": -100
+  }]
+}, {
+    "featureType": "road.highway",
+    "elementType": "geometry.fill",
+    "stylers": [{
+      "color": "#000055"
+}, {
+    "weight": 2.5
+  }]
+}, {
+    "featureType": "road.highway",
+    "elementType": "geometry.stroke",
+    "stylers": [{
+      "color": "#000000"
+}, {
+    "weight": 2
+  }]
+}, {
+    "featureType": "road.arterial",
+    "elementType": "geometry",
+    "stylers": [{
+      "color": "#FF0000"
+}, {
+    "weight": 1.8
+  }]
+}, {
+    "featureType": "road.local",
+    "elementType": "geometry",
+    "stylers": [{
+      "color": "#00FF55"
+}, {
+    "weight": 1.5
+  }]
+}];
+
+Map.setOptions("roadNetwork", {
+  "iconChange": iconChange,
+  "roadNetwork": roadNetwork
+});
+```
+
+![Road](https://github.com/TC25/earthengine-community/blob/patch-2/tutorials/customizing-base-map-styles/road.png?raw=true)
```

Comment:
I think these will break once you remove your branch, but I plan to copy the images to our Cloud Storage bucket and update the URLs anyway after merge, so all good.